React/master-detail.md

```diff
@@ -1,6 +1,6 @@
 # Marvel master-detail exercise
 
-Use the Marvel API and create a small app that:
+Use the Marvel API* and create a small app that:
```

Comment:
Maybe, just maybe, I would also add _(see Troubleshooting below)_ inside parenthesis. We usually get/memorize the first thing we read.

Reply:
Very good point. I fixed it in e610ba3.

Reply:
Thanks, very nice 💯